src/data/faq_de.json

```
@@ -883,6 +883,20 @@
                             "Dieser Bildschirm wird auf iPhones mit einer iOS-Version zwischen iOS 13.5 und (einschließlich) iOS 13.6.1 angezeigt. Auf diesen iOS-Versionen ist die benötigte Version 2 des Exposure Notification Frameworks nicht verfügbar. Deshalb muss erst auf iOS 13.7 oder neuer aktualisiert werden, um die Corona-Warn-App weiterhin verwenden zu können. Weitere Informationen finden Sie in <a href='#ios135' target='_blank' rel='noopener noreferrer'>diesem</a> FAQ Eintrag."
                         ]
                     },
+                    {
+                        "title": ["Warum wird der QR-Code von der CWA nicht gescannt?",
+                                "  Warum reagiert der QR-Code-Scanner nicht?"
+                            ],
+                        "anchor":"scan_tips",
+                        "active":true,
+                        "textblock":[
+                            "Auf einigen Geräten erkennt die Kamera den QR-Code nicht sofort und es passiert zunächst gar nichts, d.h. es wird weder der QR-Code eingelesen, noch eine Fehlermeldung angezeigt. Dieses Verhalten betrifft vorrangig einige Android-Geräte, kann jedoch auch auf iOS-Geräten auftreten.",
+
+                            "Folgende Tipps helfen oft dabei, den QR-Code zu scannen:",
+                            "<ul> <li> Auf dem QR-Code erkennen Sie 3 größere Quadrate in der Ecke: Diese müssen vollständig von der Kamera erfasst werden.</li> <li>Halten Sie die Kamera etwas weiter vom QR-Code weg, bis er in etwa die Größe einer Briefmarke hat.</li> <li>Sorgen Sie für ausreichende Kontraste und Lichtverhältnisse.</li> <li>Drehen Sie den QR-Codes um 90 Grad und versuchen Sie es erneut.</li></ul>",

```

```suggestion
                            "<ul> <li> Auf dem QR-Code erkennen Sie 3 größere Quadrate in der Ecke: Diese müssen vollständig von der Kamera erfasst werden.</li> <li>Halten Sie die Kamera etwas weiter vom QR-Code weg, bis er in etwa die Größe einer Briefmarke hat.</li> <li>Sorgen Sie für ausreichende Kontraste und Lichtverhältnisse.</li> <li>Drehen Sie den QR-Code um 90 Grad und versuchen Sie es erneut.</li></ul>",
```
Code should be singular in last sentence.

---

corrected, thanks